### PR TITLE
Add saml docs

### PR DESCRIPTION
**Pull request by pressayuliya**

### CODE REVIEW

File: src/md/pages/documentation.hbs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: Documentation
 docs: [
 
+'src/computeds/md/src/Plugins/index.md',
+    'src/computeds/md/src/Plugins/AzureSAMLIntegration.md'
```
Comment (AmsterGet) [inline src/md/pages/documentation.hbs:6]: Please add trailing comma at the end of line.

Comment (AmsterGet) [reply src/md/pages/documentation.hbs:6]: Are you sure this section should be at the top of the documentation?

**Review by AmsterGet — CHANGES_REQUESTED** [review]
Here we've added separate Plugins section to the docs structure.
Is it make sense to hold all plugins related docs under such section before publishing it to the prod?
@alena-babrova what do you think?

**Comment by alena-babrova** [comment]
> Here we've added separate Plugins section to the docs structure.
> Is it make sense to hold all plugins related docs under such section before publishing it to the prod?
> @alena-babrova what do you think?

Yes, I agree with you @AmsterGet.
I think Plugins section should include all plugins documentation.
We already have plugins documentation in another place.

image

File: src/md/pages/documentation.hbs (hunk 1 above)
Comment (AmsterGet) [reply src/md/pages/documentation.hbs]: Decided to leave integrations documentation in the current section for now.
Discussion scheduled.

**Pull request merged**